index.js

```diff
@@ -6,10 +6,12 @@ var querystring = require("querystring");
 var _ = require("lodash");
 var jwt = require("jsonwebtoken");
 var CacheService = require("cache-service");
+var fs = require("fs");
+var uuid = require('uuid');
+
+var clientVer = JSON.parse(fs.readFileSync('package.json', 'utf8')).version || "unknown";
```

Comment:
The rule for it is not configured. I wouldn't worry about adding it now - we're going to replace JSHint in future.